src/ReadVTK.jl

```diff
@@ -1032,8 +1032,12 @@ function get_cells(vtk_file::VTKFile)
   @assert !isnothing(offsets)
   @assert !isnothing(types)
 
-  # Create VTKCells container
-  VTKCells(get_data(connectivity), get_data(offsets), get_data(types))
+  # Create VTKCells container, note that connectivity must be offset by one to match WriteVTK data
+  VTKCells(
+    get_data(connectivity) + oneunit.(get_data(connectivity)),
+    get_data(offsets),
```

Comment:
I think the offsets need to be incremented by one as well, don't they?
```suggestion
    get_data(offsets) + oneunit.(get_data(connectivity)),
```

Reply:
But please check if that makes sense. I believe that the offsets should point at the last value for each cell, thus the final offset should probably be the same as the length of the points array?

Reply:
The change you suggest would be like:
```julia
get_data(offsets) + oneunit.(get_data(offsets))
```


and working through a simple example like:


```julia
points = rand(3, 5)  # 5 points in three dimensions
cells = [MeshCell(VTKCellTypes.VTK_TRIANGLE, [1, 4, 2]),
    MeshCell(VTKCellTypes.VTK_QUAD, [2, 4, 3, 5])]

vtk_grid("test/simplevtk", points, cells) do vtk
    # add datasets...
end

vtkcase = VTKFile("test/simplevtk.vtu")
points = get_points(vtkcase)
cells = get_cells(vtkcase)
```

from cells you get 
```julia
VTKCells{Vector{Int64}, Vector{Int64}, Vector{UInt8}}([1, 4, 2, 2, 4, 3, 5], [4, 8], UInt8[0x05, 0x09])
```
This offset points to 4 and 8, which the length of the connectivity vector is 7 anyway. So this seems wrong. Without offsetting the offset array I get:
```julia
VTKCells{Vector{Int64}, Base.ReinterpretArray{Int64, 1, UInt8, Vector{UInt8}, false}, Vector{UInt8}}([1, 4, 2, 2, 4, 3, 5], [3, 7], UInt8[0x05, 0x09])
```
Which seems correct, although now that I see this, I notice that the oneunit operation changes the connectivity from type ReinterpretArray to a proper Vector, though I don't think this changes again significant.

Without any changes in this PR the current output would be:
```julia
VTKCells{Base.ReinterpretArray{Int64, 1, UInt8, Vector{UInt8}, false}, Base.ReinterpretArray{Int64, 1, UInt8, Vector{UInt8}, false}, Vector{UInt8}}([0, 3, 1, 1, 3, 2, 4], [3, 7], UInt8[0x05, 0x09])
```

Reply:
I think you're right. It seems like the offset is "into the connectivity array for the end of each cell" (https://kitware.github.io/vtk-examples/site/VTKFileFormats/#unstructuredgrid), which imho is ambiguous, but which seems to be C-style. That is, it points to *one **past** the last point* in the connectivity array. I'll close this and the remark below as resolved.